src/build/patches.ts

```diff
@@ -66,14 +67,31 @@ function handleMixin(node: Node, mixins: Record<string, any>) {
   if (typeof name !== "string") {
     throw new Error("Missing mixin name");
   }
-  const rawEvents = node.children.filter(
-    (child: any) => child.name === "event",
-  );
-  const event: Event[] = rawEvents.map((child: any) => ({
-    name: child.values[0],
-    type: child.properties.type,
-  }));
-  mixins[name] = { name, events: { event } };
+
+  const event: Event[] = [];
+  const property: Properties = {};
+
+  for (const child of node.children) {
+    const name = child.values[0] as string;
+    switch (child.name) {
+      case "event":
+        event.push({
+          name,
+          type: child.properties.type as string,
+        });
+        break;
+      case "property":
+        property[name] = {
+          name,
+          exposed: child.properties?.exposed as string,
+        };
```

Comment:
I think separate functions for each is better, as I'm sure this will become longer, even for property alone.

Reply:
I think this wouldn't become super huge to justify extra modules, for now let's keep it this way and see how it goes.